File: thegamer.com/_integrations/Java/OkHttp.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
	.url("https://apitube.io/v1/sets/67017857?limit=250&offset=0")
	.get()
	.addHeader("X-ApiTube-Key", "YOUR-KEY-HERE")
	.build();

Response response = client.newCall(request).execute();

File: thegamer.com/_integrations/Kotlin/OkHttp.kt
val client = OkHttpClient()

val request = Request.Builder()
	.url("https://apitube.io/v1/sets/67017857?limit=250&offset=0")
	.get()
	.addHeader("X-ApiTube-Key", "YOUR-KEY-HERE")
	.build()

val response = client.newCall(request).execute()
